ui/src/spinbox.tsx

```diff
@@ -108,16 +111,23 @@ export const
       }, [max, min, parseValue, precision]),
       handleOnInput = React.useCallback((val: U) => {
         wave.args[name] = val
+        m.value = val
         if (trigger) wave.push()
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Any reason for not including `m` as a dependency?

Reply:
No specific reason, I just don't find it necessary. I have no problem with including it.